app/js/components/rule/ruleSummary/ruleSummary.directive.js

```diff
@@ -16,7 +16,8 @@ function RuleSummaryCtrl(
     };
     $scope.initCollapsed = false;
 
-    if ($scope.ruleFilter && $scope.ruleId && $scope.ruleId !== $scope.report.rule_id) {
+    if (($scope.ruleFilter && $scope.ruleId && $scope.ruleId !== $scope.report.rule_id) ||
+        (!$scope.ruleFilter && !$scope.ruleId)) {
         $scope.initCollapsed = true;
```

Comment:
This absolutely works and I'm fine merging it but it looks to me that the only time we want a card to not be collapsed is when the $scope.ruleId === $scope.report.rule_id. If that is the case then it would be better to just default initCollapsed to true and then we could just check that one case instead of two cases. This is super nitpicky. Just wanted to see if you noticed the same thing @jeudy100